contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -229,6 +229,66 @@ END;
 $BODY$
 LANGUAGE plpgsql STABLE;
 
+-- DATABASE_PRINCIPALS: Include Hard coded public, sys, INFORMATION_SCHEMA users
+ALTER VIEW sys.database_principals RENAME TO database_principals_deprecated_3_2_0;
+CALL sys.babelfish_drop_deprecated_object('view', 'sys', 'database_principals_deprecated_3_2_0');
```

Comment:
Better as well put this line after the create view. This is only in case that anything fail/crash during view creation

Reply:
makes sense, I've made the change. Thanks!